spellbook/models/balances/ethereum/erc721/balances_ethereum_erc721_latest.sql

```diff
@@ -2,11 +2,30 @@
         alias='erc721_latest'
         )
 }}
-SELECT distinct
-    wallet_address,
-    token_address,
-    tokenId,
+with
+    minutes as (
+        select
+            explode(
+                sequence(
+                    to_date('2015-01-01'), date_trunc('minute', now()), interval 1 minute
+                )
+            ) as minute
+    )
+
+SELECT
+    'ethereum' as blockchain,
+    d.minute,
+    b.wallet_address,
+    b.token_address,
+    b.tokenId,
+    b.num_tokens,
+    b.updated_at,
     nft_tokens.name as collection,
-    updated_at
-FROM {{ ref('transfers_ethereum_erc721_rolling_hour') }}
-LEFT JOIN {{ ref('tokens_ethereum_nft') }} nft_tokens ON nft_tokens.contract_address = token_address
+    nft_tokens.category
+FROM minutes d
+INNER JOIN {{ ref('transfers_ethereum_erc721_rolling') }} b ON (b.evt_block_time <= d.minute AND d.minute < b.next_evt)
```

Comment:
same as other comment

minor feedback: for readability, even though with inner join the results are the same, can we switch the FROM to be on {{ ref('transfers_ethereum_erc721_rolling') }} and the JOIN on minutes? as an outsider looking in for first time, seems that would flow better